client/src/components/MainRouter.jsx

```diff
@@ -22,7 +22,57 @@ class MainRouter extends React.Component {
         clanTag: 'MST',
         clanAvatar: 'https://avatars1.githubusercontent.com/u/30813487?v=4&s=200',
         description: 'This is the best clan in all the world',
-        foundedDate: 'July 2017'
+        foundedDate: 'July 2017',
```

Comment:
This is good for now but right now this is createdAt and updatedAt. I can alias this later if you prefer.